src/main/scala/de/bripkens/ha/App.scala

```diff
@@ -23,7 +23,7 @@ object App extends scala.App {
   implicit val system = ActorSystem("ha")
 
   // the AppActor gets us started from here on out
-  system.actorOf(Props(classOf[AppActor], mapper, configuration), "app")
+  system.actorOf(AppActor.props(mapper, configuration), AppActor.Name)
```

Comment:
In this case it would be the same. This is, because we're only passing the call to the `AppActor` constructor (by name). But it is possible to pass additional values to `Props`. If you want to have compile type safety for those as well you need the `props` method (since it is defined as `Props(clazz: Class[_], args: Any*)`). So it's better to create props the same way throughout a project, and the recommended way is to define Actor props using a method in the companion object.